packages/flutter_tools/test/general.shard/resident_web_runner_test.dart

```diff
@@ -139,8 +143,12 @@ void main() {
 
     verify(mockAppConnection.runMain()).called(1);
     verify(mockVmService.registerService('reloadSources', 'FlutterTools')).called(1);
+    verify(status.stop()).called(2);
```

Comment:
Why 2?

Reply:
Oh, is it because you're reusing the same mock status object?

Reply:
Yup!